diff --git a/node_modules/@sap/cds-plugin-enterprise-search/index.cds b/node_modules/@sap/cds-plugin-enterprise-search/index.cds
index f58c3ce..71a61f7 100644
--- a/node_modules/@sap/cds-plugin-enterprise-search/index.cds
+++ b/node_modules/@sap/cds-plugin-enterprise-search/index.cds
@@ -4,6 +4,7 @@ namespace sap.esh;
 type SearchResult {};
 
 @requires: 'authenticated-user'
+@path : '/odata/v4/enterprise-search/v20411'
 service EnterpriseSearchService {
 
 function ![all]() returns SearchResult;
